[2023-02-23T18:32:20.466+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.scrape_goes_data manual__2023-02-23T18:32:19.236143+00:00 [queued]>
[2023-02-23T18:32:20.473+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: metadata_scraper.scrape_goes_data manual__2023-02-23T18:32:19.236143+00:00 [queued]>
[2023-02-23T18:32:20.474+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-23T18:32:20.475+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-23T18:32:20.475+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-23T18:32:20.485+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): scrape_goes_data> on 2023-02-23 18:32:19.236143+00:00
[2023-02-23T18:32:20.499+0000] {standard_task_runner.py:55} INFO - Started process 231 to run task
[2023-02-23T18:32:20.504+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'metadata_scraper', 'scrape_goes_data', 'manual__2023-02-23T18:32:19.236143+00:00', '--job-id', '74', '--raw', '--subdir', 'DAGS_FOLDER/metadataScrapper.py', '--cfg-path', '/tmp/tmp6cmwolnx']
[2023-02-23T18:32:20.508+0000] {standard_task_runner.py:83} INFO - Job 74: Subtask scrape_goes_data
[2023-02-23T18:32:20.596+0000] {task_command.py:388} INFO - Running <TaskInstance: metadata_scraper.scrape_goes_data manual__2023-02-23T18:32:19.236143+00:00 [running]> on host a63d292ab6ab
[2023-02-23T18:32:20.666+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Charu Singh
AIRFLOW_CTX_DAG_ID=metadata_scraper
AIRFLOW_CTX_TASK_ID=scrape_goes_data
AIRFLOW_CTX_EXECUTION_DATE=2023-02-23T18:32:19.236143+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-23T18:32:19.236143+00:00
[2023-02-23T18:32:20.680+0000] {python.py:177} INFO - Done. Returned value was: Empty DataFrame
Columns: [id, product, year, day, hour]
Index: []
[2023-02-23T18:32:20.699+0000] {xcom.py:635} ERROR - Object of type DataFrame is not JSON serializable. If you are using pickle instead of JSON for XCom, then you need to enable pickle support for XCom in your *** config or make sure to decorate your object with attr.
[2023-02-23T18:32:20.702+0000] {taskinstance.py:1768} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 2297, in xcom_push
    session=session,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/session.py", line 72, in wrapper
    return func(*args, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 240, in set
    map_index=map_index,
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/models/xcom.py", line 627, in serialize_value
    return json.dumps(value, cls=XComEncoder).encode("UTF-8")
  File "/usr/local/lib/python3.7/json/__init__.py", line 238, in dumps
    **kw).encode(obj)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 176, in encode
    return super().encode(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/local/lib/python3.7/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/utils/json.py", line 170, in default
    return super().default(o)
  File "/usr/local/lib/python3.7/json/encoder.py", line 179, in default
    raise TypeError(f'Object of type {o.__class__.__name__} '
TypeError: Object of type DataFrame is not JSON serializable
[2023-02-23T18:32:20.718+0000] {taskinstance.py:1323} INFO - Marking task as FAILED. dag_id=metadata_scraper, task_id=scrape_goes_data, execution_date=20230223T183219, start_date=20230223T183220, end_date=20230223T183220
[2023-02-23T18:32:20.729+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 74 for task scrape_goes_data (Object of type DataFrame is not JSON serializable; 231)
[2023-02-23T18:32:20.757+0000] {local_task_job.py:208} INFO - Task exited with return code 1
[2023-02-23T18:32:20.780+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
